.github/workflows/build.yml

```diff
@@ -78,93 +110,193 @@ jobs:
+        #      - name: Cache Maven packages
+        #        uses: actions/cache@v2
+        #        with:
+        #          path: ~/.m2
+        #          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        #          restore-keys: ${{ runner.os }}-m2-
 
-      #      - name: Set project version
-      #        run: |-
-      #          ./mvnw -B -Dstyle.color=always \
-      #          -DnewVersion=${{ steps.version.outputs.version }} \
-      #          -DgenerateBackupPoms=false \
-      #          -Dtycho.mode=maven \
-      #          versions:set
-      #
-      #      - name: Update target definition
-      #        run: |-
-      #          ./mvnw -B -Dstyle.color=always \
-      #          -Dtycho.mode=maven  \
-      #          -P update-target-versions \
-      #          -f ./targetdefinition/pom.xml \
-      #          validate
-      #
-      #      - name: Update category
-      #        run: |-
-      #         ./mvnw -B -Dstyle.color=always \
-      #          -Dtycho.mode=maven \
-      #          -P update-category-versions \
-      #          -f ./site/pom.xml \
-      #          validate
-      #
-      #      - name: Set tycho version
-      #        run: |-
-      #          ./mvnw -B -Dstyle.color=always \
-      #          -Dtycho.mode=maven  \
-      #          -P update-category-versions \
-      #          tycho-versions:update-eclipse-metadata
-      #
-      - name: Build with Maven
+        # Eclipse specific build block. BEGIN
+        # It manages tycho versions in P2 world.
+        # Different eclipse project target / category can be in different places
+        #      - name: Set project version
+        #        run: |-
+        #          ./mvnw -B -Dstyle.color=always \
+        #          -DnewVersion=${{ steps.version.outputs.version }} \
+        #          -DgenerateBackupPoms=false \
+        #          -Dtycho.mode=maven \
+        #          versions:set
+        #
+        #      - name: Update target definition
+        #        run: |-
+        #          ./mvnw -B -Dstyle.color=always \
+        #          -Dtycho.mode=maven  \
+        #          -P update-target-versions \
+        #          -f ./targetdefinition/pom.xml \
+        #          validate
+        #
+        #      - name: Update category
+        #        run: |-
+        #         ./mvnw -B -Dstyle.color=always \
+        #          -Dtycho.mode=maven \
+        #          -P update-category-versions \
+        #          -f ./site/pom.xml \
+        #          validate
+        #
+        #      - name: Set tycho version
+        #        run: |-
+        #          ./mvnw -B -Dstyle.color=always \
+        #          -Dtycho.mode=maven  \
+        #          -P update-category-versions \
+        #          tycho-versions:update-eclipse-metadata
+        # Eclipse specific build block. END
+
+        # Phase 1 - Build and deploy to local stage (/target/stage
+      - name: Build with Maven (build, test)
```

Comment:
It have been changed in `test-workflow`. Please check last state of develop branch.

Reply:
I didn't see any changes in the last state of develop branch.